ethereum_history_api/oracles/src/fixtures/historyAPIConfig.ts

```diff
@@ -4,6 +4,7 @@ export const JS_FIXTURES_DIRECTORY = 'fixtures';
 export const CIRCLE_USDC_BALANCE_STORAGE_KEY = '0x57d18af793d7300c4ba46d192ec7aa095070dde6c52c687c6d0d92fb8532b305';
 export const UNISWAP_V3_USDC_BALANCE_STORAGE_KEY = '0x1f21a62c4538bacf2aabeca410f0fe63151869f172e03c0e00357ba26a341eff';
 export const BORED_APE_7TH_OWNER_OF_STORAGE_KEY = '0x405787fa12a823e0f2b7631cc41b3ba8828b3321ca811111fa75cd3aa3bb5add';
+export const NOUNS_9TH_OWNER_OF_STORAGE_KEY = '0x8a8dc4e5242ea8b1ab1d60606dae757e6c2cca9f92a2cced9f72c19960bcb458';
```

Comment:
What does this variable name mean?

Reply:
`OWNER_OF_9TH_NOUN_STORAGE_KEY`
Also change the one above

Reply:
Good point, fixed the name so it's not that confusing